lib/src/core/comms/secure_channel.rs

```diff
@@ -454,7 +454,7 @@ impl SecureChannel {
         }
     }
 
-    // Takes an unpadded message chunk and adds padding as well as space to the end to accomodate a signature.
+    // Takes an unpadded message chunk and adds padding as well as space to the end to accommodate a signature.
```

Comment:
Do not modified the host crate like this. Imagine they modify those comments upstream: you'll get as many conflicts to resolve for no real good reason.